content/en/blog/grpc-csharp-future.md

```diff
@@ -0,0 +1,124 @@
+NOTE: the Google.Protobuf library for C# is already written purely in C# (no native components), so having a pure C# implementation of gRPC gets rid of native components from developers' microservice stack completely
+
+
+### Why not keep Grpc.Core forever?
+
+Developing two implementations of gRPC in C# isn't free. It costs valuable resources and we believe that the engineering time would be better spent on making gRPC in C# easier to use and on adding new features (and fixing bugs of course), rather than needing to work on two different codebases that both serve the same purpose.  Also, having two separate implementations necessarily fragments the user base to some extent and splits the contributors' efforts into two. Also just the simple act of users needing to choose which of the two implementations they want to bet on comes with uncertainty and inherent risk (none of which we want for our users).
+
+By making grpc-dotnet the recommended implementation and by making the Grpc.Core implementation "maintenance only" (and eventually phasing it out), we are aiming to achieve the following goals:
+
+*   Free up engineering resources to work on better features and better usability.
+*   Unify the gRPC C# user base. This will lead to directing all community work and contributions toward a single implementation. It also removes the inherent friction caused by the user needing to choose which of the two official implementations to use.
+*   Address some of the well known pain points of Grpc.Core that would be too difficult to address by other means.
+*   Futureproof C#/.NET implementation of gRPC by staying aligned with the .NET community.
+
+
+### The plan
+
+**Phase 1: Grpc.Core becomes "Maintenance Only"**
+
+**When: Effective immediately (May 2021)**
+
+From now on, we are no longer going to provide new features or enhancements for Grpc.Core. Important bugs and security issues will continue to be addressed in a normal way.
+
+We will publish Grpc.Core releases normally, with the usual 6 weekly cadence.
+
+The releases will be based on the newest grpc C core native library build, so all new features that don't require C# specific work will also be included.
+
+**Phase 2: Grpc.Core becomes "Deprecated"**
+
+**When: 1 year from now (May 2022)**
```

Comment:
One year feels like a short amount of time. For .NET Framework users especially there isn't a GA version of Windows available that supports Grpc.Net.Client yet.

I think some more time between maintenance and deprecated would be a good.

Reply:
@JamesNK - I'll let @jtattermusch respond to your comment. I wanted to reassure you that while I've merged the PR, the blogpost won't show up on the website until we remove the draft status -- which will be after round(s) of edits :)

Reply:
We discussed this offline so I'm going to mark the conversation as resolved.